packages/office-ui-fabric-react/src/components/SwatchColorPicker/examples/SwatchColorPicker.Basic.Example.tsx

```diff
@@ -95,7 +95,11 @@ export class SwatchColorPickerBasicExample extends React.Component<any, IBasicSw
             { id: 'i', label: 'magentaPink', color: '#9b0062' },
             { id: 'j', label: 'black', color: '#000000' },
             { id: 'k', label: 'gray', color: '#7a7574' },
-            { id: 'l', label: 'gray20', color: '#69797e' }
+            { id: 'l', label: 'gray20', color: '#69797e' },
+            { id: 'm', label: 'white', color: 'white' },

```

Comment:
added some test cases to here

Reply:
aaaaaaaaaaaaand this breaks, since ssr uses these pages to test, so i had to remove these